Converters/LongConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace InvestingOak.Converters
+{
+    public class LongConverter : JsonConverter<long>
+    {
+        public override long Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+                if (long.TryParse(reader.GetString(), out long d))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6348574)